ci/Jenkinsfile-databricks

```diff
@@ -32,6 +32,7 @@ static String getSparklingVersion(props) {
 
 def startCluster(dbcVersion, sparkMajorVersion) {
     def jsonOutput = sh(script: """
+    . /envs/sw_env_python3.6/bin/activate
```

Comment:
Not sure, but would it make sense to factor out the "activation command" and use a string interpolation then? Having the same command path on so many places seems a little bit clumsy.

Reply:
I think a method could be introduced, for example `runShWithPythonEnv` or something like that (not sure about the naming), that would take a string command and then just prefix it with the activation and run with `sh`

Reply:
Thanks for the feedback, I will try @krasinski's suggestion.